_projects/bus_routing.md

```diff
@@ -0,0 +1,70 @@
+---
+# Remove the project attributes which don't apply to your project. (ie: if no issues, delete the issues section)
+
+published: false # change to true once ready to post
+
+title: "School Bus Routing Optimization"# project title inside quotes
+excerpt: "Optimize school bus routing plan using general constraint solver" # shows on project list page
+seo_description: "" # goes in project meta description
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Python"
+  - "Data analysis and visualizations skills"
+  - "Java Scala preferred"
+  - "Scala preferred"
+  - "Experience with graph algorithms a plus"
+  - "Knowledge of spatial statistics a plus"
+
+
+tags: # one tag per line – spaces are allowed in tags – use tags other posts use
+  - "python"
+  - "vehicle routing problem"
+  - "OpenTripPlanner"
+  - "Google Optimization Tools"
+
+difficulty: "easy" # easy, medium, hard – pick one
+
+mentors: # github username without the @ – example: designmatty
+  - "simonkassel"
+  - ""
```

Comment:
the other projects seem to have multiple mentors attached to them, would you and/or James be included on this one?